toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -513,46 +513,40 @@ func handleSELinux(selinuxMode imagecustomizerapi.SELinuxMode, resetBootLoaderTy
 		}
 	}
 
-	if selinuxMode != imagecustomizerapi.SELinuxModeDisabled {
-		err = updateSELinuxModeInConfigFile(selinuxMode, imageChroot)
-		if err != nil {
-			return selinuxMode, err
-		}
+	err = updateSELinuxModeInConfigFile(selinuxMode, imageChroot)
```

Comment:
Looks like `bootCustomizer.GetSELinuxMode` will only check the kernel-cmd setting for SElinux - at least if its `SELinuxModeDefault`.

In that case, and with the check on line 491, could we still end up in a case when the config and kernel-cmd are misaligned?

Reply:
Fair. Fixed.